### PR TITLE
Updating install instructions for B2B install in Cloud

### PR DESCRIPTION
**Pull request by maximbaibakov**
- Removing git add setup dev pub lib && git commit -a -m "b2b" - these files should not be added to the repo. 
- Updated instructions

## Purpose of this PR

- Update install instructions of B2B module to Magento Cloud environment  
- Avoid having unnecessary files in repository

## Affected URLs

- https://devdocs.magento.com/guides/v2.2/cloud/configure/setup-b2b.html

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=4364) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=4364) before we can accept your contribution.<br/>

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.2/cloud/configure/setup-b2b.md [hunk 1]
```diff
@@ -34,22 +34,17 @@ You need to add the module to `composer.json`. All extensions and modules must b
...
-3. Use a text editor to edit `composer.json`.
-4. In the `require` section, add the following:
+3. Run composer to install B2B module into your branch.
 
-        "magento/extension-b2b": "*"
-5. Save the file.
-6. Update composer to pull all B2B files into your branch.
-
-        composer update
+        composer require magento/extension-b2b
```
Comment (hguthrie) [inline guides/v2.2/cloud/configure/setup-b2b.md]:
```suggestion
    ```bash
    composer require magento/extension-b2b
    ```
```

File: guides/v2.2/cloud/configure/setup-b2b.md [hunk 2]
```diff
@@ -34,22 +34,17 @@ You need to add the module to `composer.json`. All extensions and modules must b
 
 1. Open a terminal application.
 2. Change to your local development environment root directory.
-3. Use a text editor to edit `composer.json`.
-4. In the `require` section, add the following:
+3. Run composer to install B2B module into your branch.
```
Comment (hguthrie) [inline guides/v2.2/cloud/configure/setup-b2b.md]:
```suggestion
1.  Install the B2B module using composer.
```

File: guides/v2.2/cloud/configure/setup-b2b.md [hunk 3]
```diff
@@ -34,22 +34,17 @@ You need to add the module to `composer.json`. All extensions and modules must b
...
-
-        composer update
+        composer require magento/extension-b2b
 
     You may be prompted to enter your Magento Authentication keys (public and private). If copying and pasting your keys, do not introduce additional spaces. Spaces could cause the following error:
 
         InvalidArgumentException - Could not find package magento/extension-b2b at any version for your minimum-stability (stable). Check the package spelling or your minimum-stability.
 
-5. Add the new B2B files to Git:
+5. Add the updated composer.json and composer.lock file to Git:
```
Comment (hguthrie) [inline guides/v2.2/cloud/configure/setup-b2b.md]:
```suggestion
1.  Add and commit the updated `composer.json` and `composer.lock` files.
```

File: guides/v2.2/cloud/configure/setup-b2b.md [hunk 4]
```diff
@@ -34,22 +34,17 @@ You need to add the module to `composer.json`. All extensions and modules must b
...
 
-        git add setup dev pub lib && git commit -a -m "b2b"
+        git add composer.json composer.lock && git commit -a -m "b2b"
```
Comment (hguthrie) [inline guides/v2.2/cloud/configure/setup-b2b.md]:
```suggestion
    ```bash
    git add composer.json composer.lock && git commit -a -m "install b2b module"
    ```
```

**Comment by maximbaibakov** [comment]
Hi @hguthrie 

Thanks that makes complete sense, I have updated the PR

Regards,
Maxim

**Comment by hguthrie** [comment]
running tests

**Pull request merged**